### DISCUSSION TITLE
[Warns]

### DISCUSSION THREAD

**Discussion by JustIanJ** [codeman1o1/TIJK-Bot#189] [Q&A] [answered]
Op hoelang staat de to tijd van 10 warns? 
Ps: wacht na je antwoord pls ff met sluiten.

**Comment by karstenzip** [comment, marked as answer]
The timeout for a warn is 10 minutes (600 seconds) as stated here:
https://github.com/codeman1o1/TIJK-Bot/blob/d9c10a36ca030f42db360bf5ffa73237791cd568/cogs/event_handler.py#L56

**Comment by JustIanJ** [comment]
thank you, you can now close this issue, i will make a pullreq
